### DISCUSSION TITLE
Why shouldContentHash not working with Parcel API?

### DISCUSSION THREAD

**Discussion by krutoo** [parcel-bundler/parcel#8357] [Q&A] [answered]
Hi, in my project i use Parcel API like this:

```js
const bundler = new Parcel({
  mode: 'production',
  entries: ['src/mobile.html', 'src/desktop.html'],
  defaultConfig: '@parcel/config-default',
  shouldContentHash: false,
  defaultTargetOptions: {
    publicUrl: 'some-string',
  },
});

return bundler.run();
```

In dist folder i have files with some hash postfix like `apple-touch-icon.2b9c3217.png`

How to disable this hashing? Also i want to make my own postfix

**Comment by mischnic** [comment, marked as answer]
It disables content hashing, not hashing:

`shouldContentHash` means that you get hashes that only depend on the Parcel configuration and the asset file paths, but not the content. So the hashes will be constant if you change some code

You'd have to create a namer plugin to remove hashes
